dashboard/src/modules/components/SidebarComponent/sideMenuOptions.js

```diff
@@ -0,0 +1,54 @@
+export const menuOptions = [
+  {
+    group: { key: "dashboard", title: "Dashboard" },
+    submenu: [
+      {
+        name: "Overview",
+        submenu: true,
+        key: "overview",
+        submenuOf: "dashboard",
+        link: "overview",
+      },
+      {
+        name: "Analysis",
+        submenu: true,
+        key: "analysis",
+        submenuOf: "dashboard",
+        link: "analysis",
+      },
+      {
+        name: "All runs",
+        submenu: true,
+        key: "all_runs",
+        submenuOf: "dashboard",
+        link: "all_runs",
+      },
+      {
+        name: "Results",
+        submenu: true,
+        key: "results",
+        submenuOf: "dashboard",
+        link: "results",
+      },
+    ],
+  },
+  {
+    group: { key: "tools", title: "Tools" },
+    submenu: [
+      {
+        name: "Search",
+        submenu: true,
+        key: "search",
+        submenuOf: "tools",
+        link: "search",
+      },
+      {
+        name: "Pbench Kibana",
```

Comment:
Is this visible on the dashboard as a side bar link named "Pbench Kibana"?  If so, then I think we should remove this.

Reply:
Removed it!